README.md

```diff
@@ -45,7 +45,7 @@ $ mongodb-migrate --host 127.0.0.1 --port 27017 --database test --migrations exa
 You can pass on the config file that you want to use by:
 
 ```
-$ mongodb-migrate --config mig.ini
+$ MONGODB_MIGRATIONS_CONFIG=mig.ini mongodb-migrate
```

Comment:
Continue with above, having a system env is a good idea, but we better have a default one.

Reply:
I was thinking of moving in a direction where if you don't specify any config, default values of connection parameters and migrations folders are considered. You only need to specify which database which I've also made mandatory in the command line args for now. There would be a need to revisit the constraint once we try to implement the overriding behavior as per above comment

Reply:
One use case will be:

1. I have a `config.ini` for production, when I run `mongodb-migrations`, it does the migrations against production
2. Before running it in production, I may want to do it locally for test, which means I will specify some parameters thru command line aruments
3. Even more, I may have 3 different environments, `local`, `staging` and `production`. This asks us to provide the ability of loading different config files